cloudbuild/nightly/scripts/python-scripts/requirements.txt

```diff
@@ -1,5 +1,5 @@
 absl-py==2.0.0
-avro==1.11.1
```

Comment:
To fix PR [#90](https://github.com/GoogleCloudDataproc/flink-bigquery-connector/pull/90)